ndc-client/src/apis/default_api.rs

```diff
@@ -61,6 +61,13 @@ pub async fn capabilities_get(
                     .header(reqwest::header::USER_AGENT, user_agent.clone());
             }
 
+            if let Some(ref bearer_token) = configuration.bearer_access_token {
+                req_builder = req_builder.header(
+                    reqwest::header::AUTHORIZATION,
+                    "Bearer ".to_owned() + bearer_token.as_str(),
+                );
```

Comment:
Consider

```suggestion
                req_builder = req_builder.bearer_auth(bearer_token.as_str());
```

See https://docs.rs/reqwest/latest/reqwest/struct.RequestBuilder.html#method.bearer_auth